manic/repository_git.py

```diff
@@ -526,21 +526,29 @@ def _status_v1z_is_dirty(git_output):
         * modified files
         * missing files
         * added files
-        * untracked files
         * removed
         * renamed
         * unmerged
 
-        NOTE: Based on the above definition, the porcelain status
-        should be an empty string to be considered 'clean'. Of course
-        this assumes we only get an empty string from an status
-        command on a clean checkout, and not some error
-        condition... Could alse use 'git diff --quiet'.
+        Untracked files are explicitly not considered for clean dirty status.
+
+        The porcelain v1 status is a list of item status seperated by
+        a null character, '\0'.
+
+        Untracked items begin with a '??'. Any other item is
+        considered to make the repository dirty.
 
         """
         is_dirty = False
-        if git_output:
-            is_dirty = True
+        status_list = git_output.split('\0')
+        for item in status_list:
+            if item.startswith('??'):
```

Comment:
Another way to do this would be to add `-uno` to the `git status` command. But I'm fine with this approach if you had a reason for doing it this way.